operations/iuf/stages/management_nodes_rollout.md

```diff
@@ -1,19 +1,28 @@
 # `management-nodes-rollout`
 
-The `management-nodes-rollout` stage performs a controlled rebuild of the management NCNs in order to reboot them to a new image and configuration. A rebuild is a reboot operation that clears the persistent OverlayFS file system on
-that node. IUF will account for the necessary minimum number of critical software instances running on the nodes to ensure the `management-nodes-rollout` stage operates without impacting software availability.
+The `management-nodes-rollout` stage performs a controlled update of management NCNs by configuring them with a new CFS configuration and/or rebuilding or upgrading them to a new image.
+A "rebuild" is a reboot operation that clears the persistent OverlayFS file system on that node.
```

Comment:
maybe add something like "Caution, all local storage will be discarded!"

Reply:
Fixed in next commit.